create/remote-pipelines-tasks-resolvers.adoc

```diff
@@ -10,22 +10,30 @@ Pipelines and tasks are reusable blocks for your CI/CD processes. You can reuse
+
+In these cases, the _resolvers_ in {pipelines-title} retrieve the pipeline, task, or `StepAction` definition from the specified source at run time.
 
 The following resolvers are available in a default installaton of {pipelines-title}:
 
-Hub resolver:: Retrieves a task or pipeline from the Pipelines Catalog available on {artifact-hub} or {tekton-hub}.
-Bundles resolver:: Retrieves a task or pipeline from a Tekton bundle, which is an OCI image available from any OCI repository, such as an OpenShift container repository.
+Hub resolver:: Retrieves a task, pipeline, or `StepAction` definition from the Pipelines Catalog available on {artifact-hub} or {tekton-hub}.
+Bundles resolver:: Retrieves a task, pipeline, or `StepAction` definition from a Tekton bundle, which is an OCI image available from any OCI repository, such as an OpenShift container repository.
+Git resolver:: Retrieves a task, pipeline, or `StepAction` definition from a Git repository. You must specify the repository, the branch, and the path.
+HTTP resolver:: Retrieves a task, pipeline, or `StepAction` definition from a remote HTTP or HTTPS URL. You must specify the URL for authentication.
 Cluster resolver:: Retrieves a task or pipeline that is already created on the same {OCP} cluster in a specific namespace.
-Git resolver:: Retrieves a task or pipeline binding from a Git repository. You must specify the repository, the branch, and the path.
-HTTP resolver:: Retrieves a task or pipeline from a remote HTTP or HTTPS URL. You must specify the URL for authentication.
 
-An {pipelines-shortname} installation includes a set of standard tasks that you can use in your pipelines. These tasks are located in the {pipelines-shortname} installation namespace, which is normally the `openshift-pipelines` namespace. You can use the cluster resolver to access the tasks.
+[NOTE]
+====
+In {pipelines-shortname} version {pipelines-version-number}, the cluster resolver does not support retrieving `StepAction` definitions.
+====
 
-[id="resolver-hub_{context}"]
-== Specifying a remote pipeline or task from a Tekton catalog
+An {pipelines-shortname} installation includes a set of standard tasks that you can use in your pipelines. These tasks definitions are located in the {pipelines-shortname} installation namespace, which is normally the `openshift-pipelines` namespace. You can use the cluster resolver to access the tasks.
 
-You can use the hub resolver to specify a remote pipeline or task that is defined either in a public Tekton catalog of link:https://artifacthub.io/[{artifact-hub}] or in an instance of {tekton-hub}.
+{pipelines-shortname} also provides a standard `StepAction` definition. You can use the HTTP resolver to access this definition.
+
+[id="resolver-hub_{context}"]
+== Specifying a remote pipeline, task, or step action from a Tekton catalog
```

Comment:
```suggestion
== Specifying a remote pipeline, task, or StepAction from a Tekton catalog
```

Reply:
The style guide does not permit us to use StepAction as a standalone noun. It can be a StepAction definition, StepAction resource, and so on, but when it's a standalone thing, it's "step action".